paasta_tools/kubernetes_tools.py

```diff
@@ -404,13 +404,22 @@ def __init__(self) -> None:
             ),
             fset=_set_disrupted_pods,
         )
+
         self.deployments = kube_client.AppsV1Api()
         self.core = kube_client.CoreV1Api()
         self.policy = kube_client.PolicyV1beta1Api()
         self.apiextensions = kube_client.ApiextensionsV1beta1Api()
         self.custom = kube_client.CustomObjectsApi()
         self.autoscaling = kube_client.AutoscalingV2beta2Api()
-        self.request = kube_client.ApiClient().request
+
+        self.api_client = kube_client.ApiClient()
+        self.request = self.api_client.request
```

Comment:
Does this get used anywhere?

Reply:
this = `self.request`

Reply:
Yes, in `kubernetes_tools.get_events_for_object`, to make a specific API call without using the OpenAPI-generated function call, because the latter doesn't support an option we want.